source/manual/reindex-elasticsearch.html.md.erb

```diff
@@ -7,139 +7,110 @@ parent: "/manual.html"
-`search:update_supertypes` to update documents in-place.  This can be
-done during working hours.
+## Reindex an Elasticsearch index
 
-## How to reindex an Elasticsearch index
+### 1. Run the reindex
 
-**Do not reindex on production during working hours except in an
-emergency.** Reindexing locks the index for writes, so content is not
-updated in the search index. See the [out-of-date search
-indices](#out-of-date-search-indices) section below if you need to run
-a reindexing during working hours. Reindexing takes around 2 hours to
-complete.
+To reindex all indices, run the `search:migrate_schema` Rake task on search-api:
 
-To reindex, you can use the following job:
+<%= RunRakeTask.links("search-api", "search:migrate_schema") %>
 
-<%= RunJenkinsJob.links("search_api_reindex_with_new_schema") %>
+To reindex a specific index, specify the `SEARCH_INDEX` environment variable:
 
-This task allows either all indices or a single index to be reindexed.
+<%= RunRakeTask.links("search-api", "SEARCH_INDEX=government search:migrate_schema") %>
 
-To monitor progress, SSH to a search box and check how many documents
-have been copied to the new index:
+To monitor progress, you can check how many documents have been copied to the
+new index:
 
-```bash
-gds govuk connect ssh -e integration search
-govuk_setenv search-api \
-bash -c 'curl "$ELASTICSEARCH_URI/_cat/indices?v"'
+```sh
+k exec deploy/search-api -- sh -c 'curl "$ELASTICSEARCH_URI/_cat/indices?v"'
 ```
 
-### Out-of-date search indices
-
-If any content has been published, updated or removed during the indexing
-job, then the search index will be out-of-date.
-
-See [Fix out-of-date search indices][fix-out-of-date-search-indices]
-for details.
-
-### Cleanup
+### 2. Clean up
 
 Reindexing does not delete the old index. This lets us switch back to the old
 index if there is a serious problem with the new one.
 
 Once you're confident that the reindexing was successful, delete the old
-(unaliased) index using the `search:clean` rake task:
+(unaliased) index using the `search:clean` Rake task:
 
 <%= RunRakeTask.links("search-api", "search:clean SEARCH_INDEX=alias_of_index_to_clean_up") %>
 
-Avoid leaving old indices around for more than a few days. If enough
-old indices hang around, we may hit space limitations and be unable to
-index new documents.
-
-#### Automatic Cleanup
-
-However,
-in the case wherein we end up with multiple copies of the same index left behind,
-we have an automated clean up task that removes any extra indexes over a given
-age:
-
-```
-rake search:timed_clean MAX_INDEX_AGE=number_of_days SEARCH_INDEX=alias_of_index_to_clean_up
-```
-
-This is running in a Jenkins job that clears any index over 7 days old, and will always leave at least one inactive index available (typically the
-  newest one created) alongside the active index for backup purposes.
+Avoid leaving old indices around, otherwise we may run into storage limitations
+and be unable to index new documents.
```

Comment:
Just an FYI - we've a card here to automate this process: https://trello.com/c/DgcJRu1j/551-automate-cleaning-of-stale-indexes-on-search-api